src/components/radio/radio.ts

```diff
@@ -128,9 +132,16 @@ export default class IgcRadioComponent extends EventEmitterMixin<
   @watch('checked', { waitUntilFirstUpdate: true })
   protected handleChange() {
     if (this.checked) {
+      if (this.getSiblings().some((r) => r.required)) {
+        this.required = true;
+      }
+
       this.getSiblings().forEach((radio) => {
         radio.checked = false;
         radio._tabIndex = -1;
+        if (this.required) {
+          radio.required = false;
+        }
       });
       this.input.focus();
```

Comment:
Do we really need this here? Since by default the tabindex is 0 and we update the rest of the siblings when a selection in the group is made this seems like a no-op

Reply:
Removing this line seems to cause a problem with setting the current radio as the activeElement